packages/frontend-shared/src/gql-components/error/BaseError.cy.tsx

```diff
@@ -69,18 +69,28 @@ describe('<BaseError />', () => {
       .should('have.attr', 'href', docsButton.configGuide.link)
     })
 
-    it('calls the retry function passed in', () => {
+    it('emits a \'retry\' event when clicked', () => {
```

Comment:
Nit, I like to just use backticks instead of escaping with `\` but either or is fine.